rs/replicated_state/src/canister_state/queues.rs

```diff
@@ -919,7 +920,10 @@ impl CanisterQueues {
             self.input_schedule = InputSchedule::default();
 
             // Trust but verify. Ensure that everything is actually set to default.
```

Comment:
... aaand one more late comment (sorry): maybe we wanna update the comment here.

Reply:
Done.

Reply:
Thank you.